Assets/MixedRealityToolkit/StandardAssets/Textures/MotionController_left_black_scaled.png.meta

```diff
@@ -3,7 +3,7 @@ guid: 03b731c99d54dfb4abde4d7d1b1b9199
 TextureImporter:
```

Comment:
Can you please revert any .Meta changes not related to your change please.  Can't see any reason you are modifying the existing MotionController meta files.  Just double check the existing one's are not touched.

Reply:
I think I'm done with that.

Reply:
Referring to the Oculus GO SDK: You need to choose the Oculus Virtual Reality SDK in the XR Settings and have the Oculus (Android) package installed, that's true, but except for a specific configuration in the Manifest, that's about all you have to do to get it into the store I think.
The "Oculus Integration" Unity package from the store provides more stuff, but the last time I used it, from the input perspective, it wasn't nearly as complete as the MRTK (gaze and pointing, cursor, beam...)